docs/security/index.md

```diff
@@ -10,12 +10,16 @@ This section provides information about the responsibility we take to provide a
 
 Every year Octopus undergoes a security review conducted by a third-party company. The latest report can be downloaded: [security-review.pdf](https://octopus.com/documents/security-review-executive-report.pdf)
 
+We often hear from customers wanting to know more about our security posture. We've performed a [self assessment against various industry standard controls](/docs/security/caiq/index.md). Feel free to use this in any vendor assessments you'd like to perform.
+
 ## Responsibility
 
 Octopus Deploy take responsibility to provide a secure and stable platform for managing your deployments.
```

Comment:
This doesn't sound quite right what about something like:

"Octopus Deploy has the responsibility to provide a secure and stable platform for managing your deployments.

Reply:
Thanks @wizedkyle I'm not planning on changing that particular part of the documentation, but it is still a valid question. I usually defer to our wordsmith @wordlee for such things - what do you think Lee?

Reply:
Mad ea change based on @wizedkyle's suggestion